WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -460,6 +458,28 @@ - (ReaderCoordinator *)readerCoordinator
                                                 readerMenuViewController:self.readerMenuViewController];
 }
 
+- (CreateButtonCoordinator *)createButtonCoordinator
+{
+    if (!_createButtonCoordinator) {
+        __weak __typeof(self) weakSelf = self;
+        _createButtonCoordinator = [[CreateButtonCoordinator alloc] init:self newPost:^{
+            [weakSelf dismissViewControllerAnimated:true completion:nil];
+            [weakSelf showPostTab];
+        } newPage:^{
+            [weakSelf dismissViewControllerAnimated:true completion:nil];
+            Blog *blog = [weakSelf currentlyVisibleBlog];
+            if (blog == nil) {
+                NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+                BlogService *blogService = [[BlogService alloc] initWithManagedObjectContext:context];
```

Comment:
This totally predates this pull request, but these two lines are repeated a few times in this file, maybe we could just add a property somewhere (e.g. a computed property in WPTabBarController+swift.swift) and call it?

Reply:
Yeah, I did consider this. I'd like to pull all of the `showTab` methods out into a Swift extension but didn't want to tackle that at the moment. I've added it to a method for now.